src/components/Terminal.js

```diff
@@ -0,0 +1,90 @@
+import * as React from "react";
+import styled from "styled-components";
+import forkIcon from "../images/svg/note.svg";
+
+const Terminal = () => {
+  return (
+    <Container>
+      <TerminalHeader>
+        <Title> Terminal </Title>
+        {/* No functionality as of right now. Just a placeholder */}
+        <ForkIcon />
```

Comment:
I think this is for copy to clipboard. This could be a really good resource for building this component and making it pretty

https://raptis.wtf/blog/gatsby-mdx-copy-code-button-with-confetti/

Reply:
Should it copy everything or just the first git command because that's what I'm unsure about

Reply:
Seem a bit overkill with prism lol but I'll change it up

Reply:
The idea of setting up a code component could be useful if we want to introduce reusability with other languages as well. i could see us having lots of use for a generic code component for use in guides and such

Reply:
Yeah I gotchu. Though the highliting is not quite as we wanted in the design for the landing page so we might not to do some custom coloring. Shouldn't be too bad thou